### PR TITLE
Show flash message when checker session data is removed

### PR DESCRIPTION
**Pull request by tyom**

### CODE REVIEW

**Comment by tyom** [comment]
test this please

**Comment by axemonkey** [comment]
FE LGTM OK :+1:

File: cla_public/apps/base/constants.py [hunk 1]
```diff
@@ -22,3 +22,9 @@
 
 HELP_FILLING_IN_FORM = zip(HELP_FILLING_IN_FORM_LABELS,
         HELP_FILLING_IN_FORM_LABELS)
+
+END_SERVICE_FLASH_MESSAGE = gettext(
```
Comment (s-block) [inline cla_public/apps/base/constants.py]: This will need to be lazy_gettext

Comment (tyom) [inline cla_public/apps/base/constants.py]: Updated.

**Comment by s-block** [comment]
Looks good

**Pull request merged**